src/components/Filters/Filters.tsx

```diff
@@ -170,16 +174,22 @@ export const Filters = ({ filters, setFilters, pages }: Props) => (
       </div>
     </div>
     <div className={paginationWrapper}>
-      <Pagination
-        pages={pages}
-        page={filters.page}
-        setPage={(page: number) => {
-          setFilters({
-            ...filters,
-            page: page
-          });
-        }}
-      />
+      <div className={paginationSelections}>
+        <Pagination
+          pages={pages}
+          page={filters.page}
+          setPage={(page: number) => {
+            setFilters({
+              ...filters,
+              page: page
+            });
+          }}
+        />
+      </div>
+      <div className={paginationText}>
+        {/* TODO: pull text dynamically */}
+        Displaying 100 of 500 comments
```

Comment:
```suggestion
        TODO: Displaying 100 of 500 comments
```

Reply:
Arguably, we shouldn't commit this if it's not working but If we do commit hardcoded text then it should be explicit that it is not real